build-aux/Dockerfile

```diff
@@ -98,10 +98,19 @@ COPY --from=golang /go/bin/* /buildroot/bin/
 ADD python python
 RUN time pip install --no-deps -e python
 
-# Run post-compile.sh
-ADD build-aux build-aux
-ADD post-compile.sh post-compile.sh
-RUN bash post-compile.sh
+# Set up /buildroot/bin/ (for the build environment)
```

Comment:
Can you provide more context in the commit message on the why? When I glance at the shell scripts its not quickly apparent that everything was directly inlined (_note it was a quick glance so I could be missing_)

Reply:
I've now added information to several of the commit messages.